### PR TITLE
add more metrics to Stackdriver sink

### PR DESCRIPTION
**Pull request by aleksandra-malinowska**
Adding more metrics to send directly to Stackdriver.

### CODE REVIEW

**Review by DirectXMan12 — CHANGES_REQUESTED** [review]
Please update the sink-owners document to add the stackdriver sink.  We should not have any sinks that are not listed in that document.

**Comment by aleksandra-malinowska** [comment]
Updated sink-owners document.

File: docs/sink-owners.md [hunk 1]
```diff
@@ -27,6 +27,7 @@ List of Owners
 | --------------- | ------------------ | -------------------| --------------------------------------------- | -------------- |
 | ElasticSearch   | :heavy_check_mark: | :heavy_check_mark: | @AlmogBaku / @andyxning / @huangyuqi          | :ok:           |
 | GCM             | :heavy_check_mark: | :x:                | @kubernetes/heapster-maintainers              | :ok:           |
+| Stackdriver     | :heavy_check_mark: | :x:                | @kubernetes/heapster-maintainers              | :new: #1538    |
```
Comment (DirectXMan12) [inline docs/sink-owners.md]: :new: means it's in development/not-yet-merged this should probably just be :ok:

**Review by DirectXMan12 — CHANGES_REQUESTED** [review]
minor nit on the sink-owners doc, afterwards I have no objections to this.

File: docs/sink-owners.md (hunk 1 above)
Comment (aleksandra-malinowska) [inline docs/sink-owners.md]: fixed

**Comment by aleksandra-malinowska** [comment]
@k8s-bot test this

**Review by DirectXMan12 — APPROVED** [review]

**Pull request merged**